plugins/languages/treesitter.nix

```diff
@@ -15,9 +15,15 @@ in
         description = "Install grammars with Nix (beta)";
       };
       ensureInstalled = mkOption {
-        type = with types; oneOf [ (enum [ "all" "maintained" ]) (listOf str) ];
-        default = "maintained";
```

Comment:
`maintained` also seems like it no longer exists?

Reply:
Yep, it no longer exists! Haven't updated this module in a while though, should get rid of "maintained" and default to "all"

Reply:
Do you not use `treesitter`? what's your alternative?

Reply:
Oh I do, just had it set to all in my personal config and didn't notice the change